### ISSUE TITLE
Building failed when installing PlugData via makepkg (Archlinux)

### ISSUE THREAD

**Issue by aik2mlj** [timothyschoen/PlugData#39]
Hi, I'm trying to add PlugData to AUR. I wrote a little PKGBUILD file where the building instructions is just the same in readme.
```shell
build () {
    cd "${srcdir}/PlugData"
    mkdir build && cd build
    cmake -DCMAKE_BUILD_TYPE=Release -G"Unix Makefiles" ..
    cmake --build .
}
```

However, since archlinux have some default `CFLAGS`/`CXXFLAGS` for `makepkg` defined in `/etc/makepkg.conf`:
```
CFLAGS="-march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions \
        -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security \
        -fstack-clash-protection -fcf-protection"
CXXFLAGS="$CFLAGS -Wp,-D_GLIBCXX_ASSERTIONS"
```
Everything is fine if I build PlugData manually. But if I built it via makepkg, these flags would be passed to the C/CXX compiler and I would get this error:
```
cc1: error: ‘-Wformat-security’ ignored without ‘-Wformat’ [-Werror=format-security]
```
I've googled for a while and it seems that this is due to some CMakeLists disabling the `-Wformat` flag..? I'm not that familiar with cmake so perhaps you can help me with that :)

**Comment by timothyschoen** [comment]
I've removed the -Wno-format flag, so it should be fixed now

**Comment by aik2mlj** [comment]
It builds fine now!

**Comment by aik2mlj** [comment]
Plus a little thought: maybe it's better for the user to decide whether to copy `.vst3` files to their `.vst3` directory? Say, AUR packages are commonly installed as system wide, meaning that `.vst3` files are copied to somewhere like `/usr/lib/vst3/` instead of the user's `$HOME/.vst3/` directory.

Hope that make sense..

**Comment by timothyschoen** [comment]
Yeah I could do that. There's a cmake flag for JUCE that will automatically copy these files over, I've enabled it because it's easy for development. I'll disable it now!

**Comment by timothyschoen** [comment]
I've disabled automatic copying now, there's a flag at the top of the cmake file, "COPY_PLUGIN_AFTER_BUILD", that you can use to reenable it if anyone wants that behaviour
